locales/templates/app.pot

```diff
@@ -315,9 +315,10 @@ msgctxt "preference_search_restore"
 msgid "Restore default search engines"
 msgstr ""
 
-#. Action for adding an additional search engine to the list
-msgctxt "preference_search_add"
-msgid "Add another search engine"
+#. Action for adding an additional search engine to the list.     The "+" is
+#. used to emphasize that this is an "add" action
+msgctxt "preference_search_add2"
+msgid "+ Add another search engine"
```

Comment:
It took me a bit to notice the "+" added at the beginning of the string, in case someone else is struggling in spotting the difference 👓